dataactbroker/handlers/fileHandler.py

```diff
@@ -1113,6 +1113,29 @@ def restart_validation(submission):
 
         return JsonResponse.create(StatusCode.OK, {"message": "Success"})
 
+    def fail_validation(self, upload_id):
+        # update all validation jobs to "ready"
+        sess = GlobalDB.db().session
+        initial_file_types = [FILE_TYPE_DICT['appropriations'], FILE_TYPE_DICT['program_activity'],
+                              FILE_TYPE_DICT['award_financial']]
+
+
+        jobs = sess.query(Job).filter(Job.submission_id == upload_id).all()
+
+        # set all jobs to their initial status of "waiting"
+        for job in jobs:
+            job.job_status_id = JOB_STATUS_DICT['waiting']
+
+        # update upload jobs to "running", only for files A, B, and C
```

Comment:
updated to set all the jobs associated with the submission to be set to failure so the submission can be deleted.